templates/server/pom.xml

```diff
@@ -37,6 +37,8 @@
           <delimiters>
             <delimiter>$[*]</delimiter>
           </delimiters>
+          <!--Requires below statement, please see https://issues.apache.org/jira/browse/MRESOURCES-190 -->
+          <addDefaultExcludes>false</addDefaultExcludes>
```

Comment:
We need to test this. The explicit `include` is not enough? Disabling the default excludes may include some other undesired files into the archetype. Maybe we can create archetype before and after this change and diff the files included.

Reply:
yes have ask @Rohit90700  also to test this at his end

Reply:
Hi @vapadwal  , I have tested ,its working fine for me.

Reply:
@Rohit90700 thanks for testing so far.
I am not in doubt that with the change `.gitignore` will be added but will also other undesired files (`*.bak`, `*.log`,`.project`, ...) be added accidentally in the archetype?
Did you diff the outcome before and after this change as suggested in my comment above?

Reply:
Hi @hohwille , I have tested both scenario before and after the change as you suggested above comment. there is only one file difference of .gitignore .